toolkit/scripts/toolchain/cross/init

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
```

Comment:
No luck with busybox here? I thought it had an init symlink.

Reply:
Actually, I didn't try out the busybox init. Will try and get back to you

Reply:
Updated to use busybox's /sbin/init. Needed to symlink /init -> /sbin/init and needed to add a custom /etc/inittab that configures the log on for what is available in the qemu virtual environment.